### PR TITLE
feat!: precise storage fees

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

This pull requests adapts new fee model introduced in GroveDB and RS Drive https://github.com/dashpay/rs-platform/pull/170

## What was done?

- Block time changed to milliseconds. Introduced `StateRepsitory#getTimeMs`
- Disabled processing fees for dry run since they are too high
- Predicted fees validation is disabled too
- Pass block information to storage components
- `StateRepsitory#storeDataContract` changed to `StateRepsitory#createDataContract` and  `StateRepsitory#updateDataContract` 
- Added epoch info to block execution context
- Moved the whole state structure creation to RS Drive
- Removed data contract cache from JS Drive in favor of RS Drive
- Fees calculation for identity is not implemented since the underling logic will be replaced soon with Identity v2.
- Default identity funding amount in SDK changed from `100000` to `1000000` since fees are higher now.

## How Has This Been Tested?

With tests

## Breaking Changes

Fees calculated different so block results and previous chain data is not compatible.
Default identity funding amount in SDK changed from `100000` to `1000000` since fees are higher now.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 1]
```diff
@@ -3051,12 +3121,17 @@ describe('DocumentRepository', function main() {
...
-            const result = await documentRepository.find(queryDataContract, documentType, {
-              where: [
-                ['a', operator, value],
-              ],
-              orderBy: [['a', 'asc']],
-            });
+            const result = await documentRepository.find(
+              queryDataContract,
+              documentType,
+
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:3127]: Extra line

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 2]
```diff
@@ -2844,7 +2909,12 @@ describe('DocumentRepository', function main() {
             ],
           };
 
-          const result = await documentRepository.find(dataContract, document.getType(), query);
+          const result = await documentRepository.find(
+            dataContract,
+            document.getType(),
+
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:2915]: Extra line

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 3]
```diff
@@ -2782,7 +2838,12 @@ describe('DocumentRepository', function main() {
...
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:2844]: Extra line

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 4]
```diff
@@ -2707,7 +2758,12 @@ describe('DocumentRepository', function main() {
             startAfter: documents[0].id,
           };
 
-          const result = await documentRepository.find(dataContract, document.getType(), options);
+          const result = await documentRepository.find(
+            dataContract,
+            document.getType(),
+
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:2764]: Extra line

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 5]
```diff
@@ -2645,6 +2695,7 @@ describe('DocumentRepository', function main() {
           const result = await documentRepository.find(
             dataContract,
             document.getType(),
+
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:2698]: Extra line

File: packages/js-drive/test/integration/document/DocumentRepository.spec.js [hunk 6]
```diff
@@ -2616,6 +2665,7 @@ describe('DocumentRepository', function main() {
...
```
Comment (shumkov) [inline packages/js-drive/test/integration/document/DocumentRepository.spec.js:2668]: Extra line

File: packages/js-drive/lib/abci/handlers/initChainHandlerFactory.js [hunk 7]
```diff
@@ -64,17 +67,17 @@ function initChainHandlerFactory(
...
-    // Create misc tree
-    await groveDBStore.createTree(
-      [],
-      Buffer.from([5]),
-      { useTransaction: true },
+    const blockInfo = new BlockInfo(
+      0,
+      0,
+      protoTimestampToMillis(time),
     );
```
Comment (shuplenkov) [inline packages/js-drive/lib/abci/handlers/initChainHandlerFactory.js:74]: We don't need to create tree here anymore?

Comment (shumkov) [reply packages/js-drive/lib/abci/handlers/initChainHandlerFactory.js:74]: Yes, now it's RS Drive's responsibility.

**Review by shuplenkov — APPROVED** [review]

**Pull request merged**